easybuild/easyconfigs/r/R-bundle/Ignore-BI_COMM_GLOBAL-on-intel.patch

```diff
@@ -0,0 +1,43 @@
+--- pbdBASE.orig/src/export_blacs/pkg_tools.c   2015-06-02 15:25:57.705288000 +0200
```

Comment:
please add a comment above explaining what this does, and who came up with it (I think you did?)